src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -359,16 +480,12 @@
                           (i18n/label :t/failed-to-fetch-community))
        :container-style {:flex 1 :justify-content :center}}]]))
 
-(defn- community-card-page-view
-  [id]
-  (let [{:keys [joined name images]
-         :as   community} (rf/sub [:communities/community id])]
-    (if community
-      [community-scroll-page id joined name images]
-      [community-fetching-placeholder id])))
-
 (defn view
   [id]
-  (let [id (or id (rf/sub [:get-screen-params :community-overview]))]
+  (let [community-id    (or id (rf/sub [:get-screen-params :community-overview]))
+        community       (rf/sub [:communities/community community-id])
+        start-expanded? (not (:joined community))]
     [rn/view {:style style/community-overview-container}
-     [community-card-page-view id]]))
+     (if community
+       [community-overview community-id start-expanded?]
+       [community-fetching-placeholder community-id])]))
```

Comment:
It's a lot of code, but hopefully now it's easier to read since it's divided in multiple separate components, any code readability suggestion is welcome